tsconfig.json

```diff
@@ -5,9 +5,8 @@
         "outDir": "./dist/out-tsc",
         "sourceMap": true,
         "declaration": false,
-        "module": "esnext",
+        "module": "es2020",
```

Comment:
the future is now...

Reply:
![image](https://user-images.githubusercontent.com/11832552/118238082-b89d0f80-b4a0-11eb-8464-76fbc006e1c6.png)